package.json

```diff
@@ -78,6 +79,7 @@
     "eslint-plugin-jest-dom": "^4.0.1",
     "eslint-plugin-svelte3": "^3.4.1",
     "eslint-plugin-testing-library": "^5.2.0",
+    "html-to-text": "^8.2.0",
```

Comment:
Don't we already have `dompurify`? How is this different?

Reply:
DOMPurify only strips dangerous tags, afaik